drop-support-for-mkdocs-0-17-3.rst

```diff
@@ -0,0 +1,36 @@
+.. post:: August 2, 2023
+   :tags: builders
+   :author: Manuel
+   :location: BCN
+   :category: Changelog
+
+Drop support for MkDocs 0.17.3
+==============================
+
+Historically, Read the Docs has installed MkDocs ``0.17.3`` by default if the project didn't define a specific version of it.
+Now, this version is a pretty old and we recommend our users to use the latest version of MkDocs. Currently, ``1.5.1``.
+
+To avoid failing builds of projects using the version installed by default if we upgrade to the latest MkDocs version available,
+we kept installing the old one during all this time.
+Now, *we are changing that behavior to always install the latest available version* and we wanted to let you know about that so you can prepare your project.
```

Comment:
```suggestion
To avoid failing builds we kept the old default version of MkDocs. *We are changing that behavior to always install the latest available version* and we wanted to let you know about that so you can prepare your project.
```